test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -930,6 +930,467 @@ test.describe( 'Navigation block', () => {
+			await page.getByRole( 'button', { name: 'Revisions' } ).click();
+			await page
+				.getByRole( 'menuitem', { name: 'Reset to defaults' } )
+				.click();
+
+			await editor.saveSiteEditorEntities();
+		} );
+
+		test( 'As a user I expect my navigation links to inherit the colors from the parent container', async ( {
+			admin,
+			page,
+			editor,
+		} ) => {
+			//Make changes to the theme link colors and make sure the group block colors are the ones we see
+			await page
+				.getByRole( 'button', { name: 'Styles', exact: true } )
+				.click();
+			await page.getByRole( 'button', { name: 'Colors styles' } ).click();
+			await page
+				.getByRole( 'button', { name: 'Color Link styles' } )
+				.click();
+			//rgba(207,46,46) is the color of the "vivid red" color preset
+			await page
+				.getByRole( 'button', { name: 'Color: Vivid red' } )
+				.click( { force: true } );
+			await page.getByRole( 'tab', { name: 'Hover' } ).click();
+			//rgba(155,81,224) is the color of the "vivid purple" color preset
+			await page
+				.getByRole( 'button', { name: 'Color: Vivid purple' } )
+				.click( { force: true } );
+			await editor.canvas.click( 'body' );
+
+			//We group the nav block and add colors to the links inside the group block
+			await editor.canvas
+				.getByRole( 'document', { name: 'Block: header' } )
+				.focus();
+			await editor.canvas
+				.getByRole( 'document', { name: 'Block: Navigation' } )
+				.click();
+			await page
+				.getByRole( 'toolbar', { name: 'Block tools' } )
+				.getByRole( 'button', { name: 'Options' } )
+				.click();
+			await page.getByRole( 'menuitem', { name: 'Group' } ).click();
+			await page
+				.getByRole( 'button', { name: 'Settings', exact: true } )
+				.click();
+			await page.getByRole( 'tab', { name: 'Styles' } ).click();
+			await page.getByRole( 'button', { name: 'Color options' } ).click();
+			await page
+				.getByRole( 'menuitemcheckbox', { name: 'Show Link' } )
+				.click();
+			await page.getByRole( 'tab', { name: 'Styles' } ).click();
+			await page
+				.getByRole( 'button', { name: 'Color Link styles' } )
+				.click();
+			//rgba(0,208,132) is the color of the "Vivid green cyan" color preset
+			await page
+				.getByRole( 'button', { name: 'Color: Vivid green cyan' } )
+				.click( { force: true } );
+			await page.getByRole( 'tab', { name: 'Hover' } ).click();
+			//rgba(255,105,0) is the color of the "Luminous vivid orange" color preset
+			await page
+				.getByRole( 'button', { name: 'Color: Luminous vivid orange' } )
+				.click( { force: true } );
+			await editor.canvas.click( 'body' );
+
+			const firstLink = editor.canvas
+				.locator( 'a' )
+				.filter( { hasText: 'First Link' } );
+			await editor.canvas
+				.getByRole( 'document', { name: 'Block: header' } )
+				.focus();
+			await editor.canvas
+				.getByRole( 'document', { name: 'Block: Navigation' } )
+				.click();
+
+			//Expect the first link to inherit the link color from the parent group block
```

Comment:
Super nit (😸 ) - ditto for all instances. 

```suggestion
			// Expect the first link to inherit the link color from the parent group block
```

Reply:
I suppose it's not in the linting rules! fixed